Sources/MessagingInApp/Gist/Managers/InlineMessageManager.swift

```diff
@@ -18,9 +18,11 @@ protocol InlineMessageManagerDelegate: AnyObject {
  */
 class InlineMessageManager: MessageManager {
     var inlineMessageView: GistView? {
-        let view = super.gistView
-        view?.delegate = self
-        return view
+        if super.gistView.delegate == nil {
+            super.gistView.delegate = self
+        }
+
+        return super.gistView
```

Comment:
See [these notes](https://github.com/customerio/customerio-ios/pull/756/files#diff-70b8fb68d815f2ebfa219cf5e9332ad86a7b2081a7a87e6dbcc43c0ddd850e4aR22) to explain this change.